newapi/ooni_api_uploader.py

```diff
@@ -95,7 +86,7 @@ def fill_postcan(hourdir, postcanf):
     measurements = []
     postcan_byte_thresh = 20 * 1000 * 1000
     # Open postcan
-    with tarfile.open(str(postcanf), "w") as tar:
+    with tarfile.open(str(postcanf), "w:gz") as tar:
```

Comment:
What has been done with the tarfile that were previously not compressed? Have they been recompressed or do we handle when reading them the fact they might be gzipped or not? If the later, how do we distinguish between the two?

Reply:
This PR changes only the uploader and this codebase has no tools to reprocess old tarballs or decompress them.
The fastpath should autodetect compression.